test-support/helpers/ember-i18n/test-helpers.js

```diff
@@ -1,15 +1,15 @@
 /* globals QUnit, expect */
 
-import Ember from 'ember';
+import { registerHelper } from '@ember/test';
+import { _t } from 'ember-i18n/test-support/helpers';
```

Comment:
I think this will be an issue for people that don't have `@ember/test-helpers` yet, as the import above will fail. Either you move `_t()` to a module that has no dependency on `@ember/test-helpers` or you import it using intimate API (`require()`) and wrap that in a try-catch

Reply:
You're right, that will probably fail. I'll move it.